### ISSUE TITLE
About Real Project vs Test

### ISSUE THREAD

**Issue by Subra730** [cganesh8/New-idea#4]
@cganesh8 That's fine, let's move on with your idea.

**Comment by cganesh8** [comment]
@Subra730, yes is there something you're thinking of improving your current website maybe or creating something new in your business. Maybe that will be a real tangible business case to work on.
